Sources/ApolloCodegenLib/Templates/RenderingHelpers/GraphQLType+Rendered.swift

```diff
@@ -173,4 +177,47 @@ extension GraphQLNamedType {
 
     return swiftName
   }
+
+  fileprivate func qualifiedRootTypeName(
+    in context: GraphQLType.RenderContext,
+    replacingNamedTypeWith newTypeName: String? = nil,
+    config: ApolloCodegenConfiguration
+  ) -> String {
+    lazy var typeName = { newTypeName ?? self.swiftName.firstUppercased }()
+
+    lazy var schemaModuleName: String = {
+      switch self {
+      case is GraphQLCompositeType:
+        return ""
+
+      case let scalar as GraphQLScalarType where scalar.isSwiftType:
+        return ""
+
+      default:
+        switch context {
+        case .testMockField, .inputValue:
```

Comment:
`renderedAsTestMockField` doesn't look like it flows through this new logic - should it?